bin/git-bc

```diff
@@ -20,10 +20,10 @@ if (opts.indexOf('-h') > -1) {
   gitBc.on('success', log.success);
 
   // Exits program execution on ESC keypress
-  process.stdin.on('keypress', function (ch, key) {
-    if (key && key.name === 'escape') {
+  process.stdin.on('keypress', function(ch, key) {
```

Comment:
oh, this line got changed for no reason at all :disappointed: plus it breaks my standard of always having a space between `function` declaration and parenthesis :)

other than that LGTM :+1: